src/components/ImageView/index.js

```diff
@@ -279,7 +287,6 @@ class ImageView extends PureComponent {
                             styles.w100,
                         ]} // Hide image until finished loading to prevent showing preview with wrong dimensions.
                         resizeMode={Image.resizeMode.contain}
-                        onLoadStart={this.imageLoadingStart}
```

Comment:
Why this is removed?

Reply:
I'm adding them back and making sure loadingStart doesn't call setState too many times from that function